huaweicloud/resource_huaweicloud_api_gateway_api.go

```diff
@@ -26,6 +26,11 @@ func resourceAPIGatewayAPI() *schema.Resource {
 		},
 
 		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
```

Comment:
`ForceNew` must be true when `region` has changed for resource.

Reply:
Got it, I will push a patch and fix this.

Reply:
Fix compeleted.